packages/office-ui-fabric-react/src/components/pickers/examples/TagPicker.Basic.Example.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { TagPicker } from 'office-ui-fabric-react/lib/components/pickers/TagPicker/TagPicker';
```

Comment:
No, it's fine because that module actually exists. For the checkbox case, we aren't resolving the module at Checkbox/index.